rclcpp/include/rclcpp/experimental/subscription_intra_process.hpp

```diff
@@ -153,6 +153,7 @@ class SubscriptionIntraProcess : public SubscriptionIntraProcessBase
   {
     rmw_message_info_t msg_info;
     msg_info.from_intra_process = true;
+    msg_info.publisher_gid = {0, {0}};
```

Comment:
Nitpick: I would suggest to initialize the member in the order they appear in the struct. So `publisher_gid` before `from_intra_process`.

Reply:
done 9c4d0db